test/versioned/otel-bridge/logs.test.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const test = require('node:test')
+const assert = require('node:assert')
+const stream = require('node:stream')
+
+const helper = require('../../lib/agent_helper')
+
+test('otel decorated logs do not overwrite NR data', (t, end) => {
+  process.env.OTEL_BLRP_SCHEDULE_DELAY = 1_000 // Interval for processor to ship logs
+
+  const agent = helper.instrumentMockedAgent({
+    opentelemetry_bridge: {
+      enabled: true,
+      logs: { enabled: true }
```

Comment:
you need to also enable traces in otel bridge at the moment. this is because the context manager and trace propagator are registered there. The reason you aren't seeing any logs on tx is because of the lack of assigning the global context manager